index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+
+</head>
+
+
+<body>
+    <main id = "main">
+        <h1 id = "title">Youssef and Gustavo</h1>
+        <p> Master Yoda: "Do... or do not, There is no try"</p>
+        <div id = "img-div">
+            <img id="image" src="picture.jpg" alt="">
+        </div>
+        <div id = "img-caption">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Possimus iusto officiis ullam tenetur est libero dolorem vel 
+                rerum vitae porro, architecto dolore, iure voluptates quo. Explicabo ipsa ut mollitia eaque.</p>
+
```

Comment:
I would like to advise you to consider using the closing tag for all paragraphs, in the same column indentation, for more ordered and clean code.